### PR TITLE
fix nette utils 2.3.10 - deprecated method Html::add()

### PR DESCRIPTION
**Pull request by janatjak**

### CODE REVIEW

**Comment by davidindra** [comment]
Why to do it this so complicated way? Why not just substitute `add()` => `addHtml()`? :)
Look at my [PR](https://github.com/Kdyby/HtmlValidatorPanel/pull/5).

**Comment by janatjak** [comment]
If you rename add() to addHtml(), it will be "BC break", because nette\utils <=2.3.9 doesn't contain method addHtml().

**Comment by simPod** [comment]
I didn't check comments on https://github.com/Kdyby/HtmlValidatorPanel/pull/6. I'd like to merge this, can you just resolve conflict please? Thanks! Will release this as a new version.

**Comment by janatjak** [comment]
@simPod done

File: ValidatorPanel.php [hunk 1]
```diff
@@ -794,9 +794,15 @@ class ValidatorPanel extends Nette\Object implements Tracy\IBarPanel
...
 	{
-		return Html::el('span', ['title' => 'HTML Validator'])
-			->addHtml(Html::el()->setHtml(file_get_contents(__DIR__ . '/html_icon.svg')))
-			->addHtml(Html::el('span', ['class' => 'tracy-label'])->setText($this->errors ? (count($this->errors) . ' problems') : 'OK'));
+        $el = Html::el('span', ['title' => 'HTML Validator']);
+
+        // fix nette utils 2.3.10 - deprecated method Html::add()
+        $callMethod = method_exists($el, 'addHtml') ? 'addHtml' : 'add';
+
+        return $el
```
Comment (simPod) [inline ValidatorPanel.php]: move creating HTML elelement below `$callMethod`. Better to have it together

return Html::el('span', ['title' => 'HTML Validator'])

**Comment by janatjak** [comment]
@simPod done

File: ValidatorPanel.php [hunk 2]
```diff
@@ -794,9 +794,13 @@ class ValidatorPanel extends Nette\Object implements Tracy\IBarPanel
...
 	{
+		// fix nette utils 2.3.10 - deprecated method Html::add()
+		$callMethod = method_exists(Html::class, 'addHtml') ? 'addHtml' : 'add';
+
 		return Html::el('span', ['title' => 'HTML Validator'])
-			->addHtml(Html::el()->setHtml(file_get_contents(__DIR__ . '/html_icon.svg')))
-			->addHtml(Html::el('span', ['class' => 'tracy-label'])->setText($this->errors ? (count($this->errors) . ' problems') : 'OK'));
+			->$callMethod(Html::el()->setHtml(file_get_contents(__DIR__ . '/html_icon.svg')))
+			->$callMethod(Html::el('span', ['class' => 'tracy-label'])
+				->setText($this->errors ? (count($this->errors) . ' problems') : 'OK'));
```
Comment (simPod) [inline ValidatorPanel.php]: fix indentation please

**Comment by simPod** [comment]
@janatjak Thank you

**Pull request merged**